### PR TITLE
fix(ui5-breadcrumbs): remove text-decoration from separator

### PR DESCRIPTION
**Pull request by hardl**
Remove underline text decoration shown on mouse hover from separator.

Before:
image

After:
image

### CODE REVIEW

**Comment by nnaydenow** [comment]
Hi @hardl,

Could you please add some background about the issue that we are fixing with this PR?

Regards,
Nayden

**Comment by hardl** [comment]
@nnaydenow sure, sorry, I didn't expect this PR already getting picked up by someone since it is in draft state ;) 
I have added some description. Let me know if you need more info.

**Comment by petermuessig** [comment]
Hi @nnaydenow, hi @kineticjs, hi @ilhan007,

the issue has been discovered by a designer that the breadcrumb also shows underlines for the breadcrumb separators. This is most probably not the expected behavior as only the links should be underlined. 

Best regards,
Peter

**Comment by ilhan007** [comment]
Hi @hardl @petermuessig, @nnaydenow and his team are the owners of the Breadcrumbs component and he is already looking into it. It seems like a bug, so I expect to merge it in 1-2 days. @hardl thank you for your contribution!

File: packages/main/src/themes/Breadcrumbs.css [hunk 1]
```diff
@@ -62,13 +62,14 @@
 }
 
 /* links separator */
-[ui5-link]::after {
+.ui5-breadcrumbs-link-wrapper::after {
```
[the remark this answers is not in the document]
Comment (hardl) [reply packages/main/src/themes/Breadcrumbs.css:65]: Hi @kineticjs ,
you are absolutely right!
Maybe introducing a dedicated class for that (or an attribute) could save us some lines of code (wrt separator styles).
Do you have any conventions here?

[the remark this answers is not in the document]
Comment (hardl) [reply packages/main/src/themes/Breadcrumbs.css]: committed a proposal, let me know what you think

[the remark this answers is not in the document]
Comment (kineticjs) [reply packages/main/src/themes/Breadcrumbs.css]: Looks good to me
@nnaydenow are you fine with the approach to use an attribute to mark the list items that should have a separator?
If we keep the attribute name and boolean type as currently suggested, we can, as you know, also omit the value to make it even more concise

[the remark this answers is not in the document]
Comment (nnaydenow) [reply packages/main/src/themes/Breadcrumbs.css]: Hi @hardl , @kineticjs ,

I rechecked other component templates and as I saw we mark elements like list items with CSS class. I would prefer to use this solution to be consistent with other component implementation.

Regards,
Nayden

[the remark this answers is not in the document]
Comment (kineticjs) [reply packages/main/src/themes/Breadcrumbs.css]: Ok, so it should be a class name of "ui5-breadcrumbs-xxx" format

I myself can't come up with a good proposal so far (I thought of e.g. ui5-breadcrumbs-item-separator and ui5-breadcrumbs-nav-history-item but all seem imperfect )

btw another simple option would be to not add any new class and instead use li:not(.ui5-breadcrumbs-current-location)
selector 

I'm open to other proposals as well...
Thnx

[the remark this answers is not in the document]
Comment (hardl) [reply packages/main/src/themes/Breadcrumbs.css]: Actually I like the simple solution with li:not(.ui5-breadcrumbs-current-location).
Changes are in.
@nnaydenow what do you think?

**Review by kineticjs — APPROVED** [review]
@nnaydenow please have a final look... o/w I think this PR is good to merge

**Review by ilhan007 — APPROVED** [review]

**Review by nnaydenow — APPROVED** [review]

**Pull request merged**